data/human/human missions.txt

```diff
@@ -7453,3 +7453,442 @@ mission "Adelita 5"
+			choice
+				`	"Sure thing, I'll see you later."`
+					goto next2
+				`	"You have two offices?"`
+					goto offices
+				`	"I don't care about prying eyes! What do you want and what do I get?"`
+			`	Howard raises an eyebrow and then waves you off with a chuckle. "All in good time, my friend," he says, leaning back. "It's nothing you can't handle. I'll fill you in once we're safely on Sunracer, and let's just say it'll be worth your while."`
+				goto next2
+			label offices
+			`	Howard leans back with a knowing smile. "Well, 'office' is a bit generous for this place," he admits, gesturing around the room. "This is more of a glorified conference room that I can reserve for investor meetings here on Hephaestus. My real office is back on Sunracer, where the serious deals happen. You'll see what I mean when we get there."`
+			choice
+				`	"Can't wait."`
+					goto next2
+				`	"Is that a glorified conference room too?"`
+			`	Howard's smile falters slightly. He forces a light chuckle, though the tension is noticeable. "Well, if I'm being honest, the Sunracer office is... also technically reservable," he admits, rubbing his neck and looking almost pained. "But just as nice! And, uh... look, my real office is technically a cubicle, but compared to the kiosk last time we met? It's a big step up." He shrugs, trying to brush it off. "But hey, we don't need fancy spaces to make serious deals. Let's get back on track, yeah?" He clears his throat.`
+			choice
+				`	"Absolutely."`
+					goto next2
+				`	"Howard, a promotion's a big deal. The office doesn't matter--your work speaks for itself."`
+					goto kindy
+				`	"Howard, your cubicle is still pathetic. Now, let's get back on track."`
+			action
+				"syndicate business: mean to Howard" ++
+			`	Howard flinches, his jaw tightening for a moment before he forces a thin smile. "Pathetic or not, it gets the job done," he says quietly, avoiding your gaze as he fidgets with a paperweight on his desk. "Spaceport. Later." His voice is more clipped now, and he doesn't wait for a response before focusing on his work calls, shutting you out entirely.`
+				decline
+			label kindy
+			action
+				"syndicate business: mean to Howard" --
+			`	"Thanks, that means a lot. Now, I've got to get back to work."`
+			label next2
+			`	Howard's attention shifts away from you as he begins taking a series of work calls, his tone shifting seamlessly from charming to authoritative as he negotiates and makes deals. He mouths two words to you: "Spaceport... later..." It's clear that, for now, Howard is finished with you and you'll get no more information.`
```

Comment:
You've got two instances of the beginning "shift" here. Maybe you could change one of them, perhaps to this?
```suggestion
			`	Howard's attention shifts away from you as he begins taking a series of work calls, his tone changing seamlessly from charming to authoritative as he negotiates and makes deals. He mouths two words to you: "Spaceport... later..." It's clear that, for now, Howard is finished with you and you'll get no more information.`
```